### DISCUSSION TITLE
Announcing the New Era of 'open-webui': A Big Thank You to Our Community!

### DISCUSSION THREAD

**Discussion by tjbck** [open-webui/open-webui#764] [Announcements]
Hey everyone! 👋

We've got some fantastic news to share! After a lot of back-and-forth and some friendly competition, we've finally settled on a new name for our project: **'open-webui'**. It was a close race, with lots of great suggestions, but in the end, **'open-webui'** came out on top.

### A New Beginning, Same Commitment 🚀
This is more than just a name change for us; it's a new beginning. The choice of "Open-WebUI" reflects our commitment to not limiting ourselves to just LLMs. As we venture further into the realm of Artificial Intelligence, we plan to incorporate a variety of AI technologies into the user interface. This evolution will enable us to broaden our horizons and explore new, innovative ways to integrate AI into our daily digital experiences.

While this is a significant milestone, we want to reassure you that it's primarily a name change. Our core mission remains steadfast: to continue supporting, refining, and enhancing our webui. We are dedicated to making Open-WebUI the best open-source user interface for AI out there. Expect to see new features, improved functionality, and even more user-friendly designs in the upcoming releases.

### Gratitude to Our Pillars of Support 🙏
This journey wouldn't have been possible without our incredible sponsors, contributors, and users. Your unwavering support, valuable feedback, and active participation have been the cornerstone of our progress. You've been instrumental in bringing 'open-webui' to life, and for that, we are profoundly grateful.

### Let's Keep the Momentum Going 🤝
We're pumped for what's ahead, and we want you to be a part of it. Keep an eye out for updates, share your ideas, and get involved with the 'open-webui' project. Together, let's push the boundaries of what's possible with AI and Open-WebUI.

Thanks again for being awesome and joining us on this exciting journey with 'open-webui'!

Warmest Regards,
The open-webui Team

-------
**P.S.**: To the supporters of 'dog3ai/ui', the second runner up in our poll, we appreciate your enthusiasm for that name as well. Keep an eye out, as we plan on creating a forked version of 'open-webui' with a delightful Doge-themed interface. Stay tuned for updates!

**Comment by itsbrex** [comment]
Thank you for your contribution to OSS 🙏

🚀 LFG
